lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -62,11 +81,7 @@ function checkBucketAcls(bucket, requestType, canonicalID) {
         }
     }
 
-    if (requestType === 'bucketDelete' && bucket.getOwner() === canonicalID) {
```

Comment:
this can actually be removed since we have this condition https://github.com/scality/cloudserver/blob/32227f6f43e6b90ddba596e439b26d03f765ccbc/lib/api/apiUtils/authorization/permissionChecks.js#L15C3-L15C3

Reply:
for the test it's already here : https://github.com/scality/cloudserver/blob/32227f6f43e6b90ddba596e439b26d03f765ccbc/tests/unit/auth/permissionChecks.js#L22

Reply:
Right, it was a dead code, looks good